src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.hpp

```diff
@@ -1212,6 +1212,7 @@ void GenericThreadStackManagerImpl_OpenThread<ImplClass>::_ErasePersistentInfo(v
     ChipLogProgress(DeviceLayer, "Erasing Thread persistent info...");
     Impl()->LockThreadStack();
     otThreadSetEnabled(mOTInst, false);
+    otIp6SetEnabled(mOTInst, false);
```

Comment:
It doesn't hurt to add it, but I'm surprised a packet can still be received after disabling thread. I might be unaware of some underlying process.

Reply:
@jmartinez-silabs `otThreadSetEnabled` itself does not disable the radio, so yeah, `ifconfig down` helps here.

Reply:
Thank you both for the explanation